frame/support/src/traits/dispatch.rs

```diff
@@ -236,23 +236,6 @@ pub trait UnfilteredDispatchable {
 	fn dispatch_bypass_filter(self, origin: Self::Origin) -> DispatchResultWithPostInfo;
 }
 
-/// Type that can be dispatched with an additional storage layer which is used to execute the call.
-pub trait DispatchableWithStorageLayer {
-	/// The origin type of the runtime, (i.e. `frame_system::Config::Origin`).
-	type Origin;
-
-	/// Same as `dispatch` from the [`frame_support::dispatch::Dispatchable`] trait, but
-	/// specifically spawns a new storage layer to execute the call inside of.
-	fn dispatch_with_storage_layer(self, origin: Self::Origin) -> DispatchResultWithPostInfo;
-
-	/// Same as `dispatch_bypass_filter` from the [`UnfilteredDispatchable`] trait, but specifically
-	/// spawns a new storage layer to execute the call inside of.
-	fn dispatch_bypass_filter_with_storage_layer(
-		self,
-		origin: Self::Origin,
-	) -> DispatchResultWithPostInfo;
-}
-
```

Comment:
This needn't have been removed.
Was it removed in the substrate origin implementation?